README.md

```diff
@@ -384,3 +386,48 @@ const assert = v.assert(v.range([-10, 10]));
 assert(4); // pass
 assert(-100); // fail
 ```
+
+
+## Custom validators
+One of the primary goals of Fusspot is to be customizable out of the box. There are multiple ways to which one can create a custom validator. 
+
+### Simple
+A simple custom validator is nothing but a function which accepts the input `value` and returns a `string` if the input `value` doesn't pass the test. The `string` returned would then be used to show a helpful error to users. Below is an example of a path validator for node environment.
+
+```javascript
+const path = require('path');
+
+function validateAbsolutePaths(value) {
+  if (typeof value !== 'string' || !path.isAbsolute(value)) {
+    return 'absolute path';
+  }
+}
+
+const assert = v.assert(validateAbsolutePaths);
+assert('../Users'); // fail
+// Error: value must be an absolute path.
+assert('/Users'); // pass
+```
+
+### Customizing the entire error message 
+To customize the entire error message, a validator can return a function. This function gets called with an object containing
+ - **path**: This is an array containing the location error. If an error is in 
```

Comment:
Not sure what "location error" is. And looks like a sentence got cut off?

Reply:
I tried to word it differently in the latest commit, let me know if there is a better wording.